tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -351,11 +351,16 @@ def test_sharing(self, cluster, friend_account_logged_in_docker_cluster_pk_ssh):
         resource_class_name = cluster.config().get("resource_type").capitalize()
         config = cluster.config()
 
-        expected_creds = (
-            "ssh-sky-key"
-            if isinstance(cluster._creds, SSHSecret)
-            else f'{config["name"]}-ssh-secret'
-        )
+        default_keypair = rh.globals.configs.get("default_keypair")
```

Comment:
fyi it's named `default_ssh_key` currently but maybe `default_keypair` sounds better 🤔